### DISCUSSION TITLE
The worst UI ever in GitHub history.

### DISCUSSION THREAD

**Discussion by mrkrabs206** [community/community#60409] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi GitHub, Why is your new UI so bad? Like, see it's just white everywhere, no more black on the top. I'm not interested in the so-called **"For you"** feed of yours. And also, the worst UI update ever made in the **ENTIRE** history of GitHub. Like, there's not even a button called **"Go back to old GitHub".** It's trying to become GitHub a social media at this point because of what you just did. Please give me old GitHub and for everyone, **"Go back to old GitHub"** button. Thank you all.
![worst github ui ever](https://github.com/community/community/assets/110676833/51ba207e-93e7-496f-af11-2ea55b8746ee)

**Comment by GhostOf0days** [comment, marked as answer]
I honestly think people are over-exaggerating how bad it is in discussions. It's not that bad tbh.

**Reply by mrkrabs206** [reply]
I hate that for me.

**Comment by ronakjkothari** [comment]
I'm honestly a fan.

**Reply by GhostOf0days** [reply]
Same.

**Reply by mrkrabs206** [reply]
I don't care about your opinions.

**Comment by mrkrabs206** [comment]
Please don't send hate. We are trying to save the old GitHub.

**Reply by GhostOf0days** [reply]
We aren't sending hate. Simply giving our opinions. I genuinely believe the new UI is not bad.
